Cargo.toml

```diff
@@ -27,8 +27,8 @@ serde_repr = "0.1.20"
 tokio = { version = "1.44.1", features = ["fs", "rt", "macros"] }
 
 # workspace-local
-zinnia_runtime = { version = "0.21.1", path = "./runtime" }
-zinnia_libp2p = { version = "0.21.1", path = "./ext/libp2p" }
+zinnia_runtime = { version = "0.22.0", path = "./runtime" }
+zinnia_libp2p = { version = "0.22.0", path = "./ext/libp2p" }
```

Comment:
Ditto. I think you should revert all of these version changes for local packages in this monorepo. They will conflict with the changes made on the `main` branch.

Same comment applies to other `Cargo.toml` in this repository.